### DISCUSSION TITLE
Navigator enhancement

### DISCUSSION THREAD

**Discussion by SamusFairchild** [elementor/elementor#18165] [Developers Community]
This is probably not a very important thing, and doesn't really affect the functionality of elementor in general... it's just a small quality of life thing.

When editing in Elementor, it would be super nice to be able to recursively expand the elements in the Navigator....or just to have an expand/collapse all option, even. I can't imagine it'd be a big deal, but it's one of those small things that would just make it so much more convenient.

**Comment by rami-elementor** [comment]
Hi @SamusFairchild ,

The navigator panel already has the option to expand/collapse all the elements.

![image](https://github.com/elementor/elementor/assets/92088692/2e251cf5-9280-4278-88a5-b57dba866464)
